python/tvm/runtime/ndarray.py

```diff
@@ -218,6 +218,8 @@ def numpy(self):
             dtype = str(t)
         if dtype == "int4":
             dtype = "int8"
+        if dtype == "bfloat16":
+            dtype = "uint16"
```

Comment:
Would it make sense to use float16 or float32 here?

Reply:
the expected type in the onnx output is `uint16`, which is why I matched that here